src/main/kotlin/no/nav/familie/ef/sak/api/avstemming/AvstemmingController.kt

```diff
@@ -1,20 +1,32 @@
 package no.nav.familie.ef.sak.api.avstemming
 
-import no.nav.familie.ef.sak.repository.domain.Stønadstype
-import no.nav.familie.ef.sak.task.GrensesnittavstemmingTask
+import no.nav.familie.ef.sak.repository.KonsistensavstemmingRepository
+import no.nav.familie.ef.sak.repository.domain.Konsistensavstemming
+import no.nav.familie.ef.sak.service.AvstemmingService
 import no.nav.familie.kontrakter.felles.Ressurs
 import no.nav.familie.prosessering.domene.Task
 import no.nav.security.token.support.core.api.ProtectedWithClaims
 import org.springframework.web.bind.annotation.*
 
 @RestController
-@RequestMapping("/api/grensesnittavstemming")
+@RequestMapping("/api/avstemming")
 @ProtectedWithClaims(issuer = "azuread")
-class AvstemmingController (private val grensesnittavstemmingTask: GrensesnittavstemmingTask) {
+class AvstemmingController(
+        private val avstemmingService: AvstemmingService,
+        private val konsistensavstemmingRepository: KonsistensavstemmingRepository,
+) {
 
-    @PostMapping("{stønadstype}")
-    fun utløsGrensesnittavstemming(@PathVariable stønadstype: Stønadstype, @RequestBody grensesnittavstemmingDto: GrensesnittavstemmingDto): Ressurs<Task> {
-        return Ressurs.success(grensesnittavstemmingTask.utløsGrensesnittavstemming(fraDato = grensesnittavstemmingDto.fraDato, stønadstype = stønadstype, triggerTid = grensesnittavstemmingDto.triggerTid))
+    @PostMapping
+    fun utløsGrensesnittavstemming(@RequestBody avstemmingDto: AvstemmingDto): Ressurs<Task?> {
```

Comment:
`utløsGrensesnittavstemming` brukes denne for både grensesnitt og konsistens - burde den endre navn ? 

`utløs` ? Opprett?